Assets/HoloToolkit/Input/Scripts/OnGazeEnterEvent.cs

```diff
@@ -1,21 +0,0 @@
-﻿using UnityEngine;
```

Comment:
@stbertou Just so you know, these changes will most likely not make it through the PR, but it is something I'm looking at improving in the future.